<launch>

    <group>
        <!-- PCL Manager -->
        <node pkg="nodelet" type="nodelet" name="box_filter_pcl_manager" args="manager" output="screen"/>
        <!-- Run a passthrough filter to delimit in z direction -->
        <node pkg="nodelet" type="nodelet" name="psz" args="load pcl/PassThrough box_filter_pcl_manager"
              output="screen">
            <remap from="~input" to="/camera/depth_registered/points"/>
            <remap from="~output" to="/camera/depth_registered/points_z_filtered"/>
            <rosparam>
                filter_field_name: z
                filter_limit_min: 0.22
                filter_limit_max: 0.75
            </rosparam>
        </node>
    </group>
    <param name="pc_topic" value="/camera/depth_registered/points_z_filtered" />
</launch>